Sources/Internal/OperationsManager.swift

```diff
@@ -18,6 +18,18 @@ import Foundation
 class OperationsManager {
   private var grpcClient: GrpcClient
 
+  private let queryRefAccessQ = DispatchQueue(
+    label: "firebase.dataconnect.queryRef.AccessQ",
+    autoreleaseFrequency: .workItem
+  )
+  private var queryRefs = [AnyHashable: any ObservableQueryRef]()
+
```

Comment:
Both `queryRefs` and `mutationRefs` seem to grow unbounded. Is there any consideration to flushing stale entries?

Reply:
Good point. Created an issue to track this -> b/369489483